src/js/index.js

```diff
@@ -0,0 +1,110 @@
+var itemTemplate = require("../templates/item.hbs");
+let menu = JSON.parse(localStorage.getItem("savedMenu"));
+let leftPrice = document.querySelector('#left-price');
+let rightPrice = document.querySelector('#right-price')
+let currentCategory = '';
+
+function renderMenu(items) {
+  let itemsContainer = document.querySelector('#handle-test');
+  if (itemsContainer !== undefined) {
+    itemsContainer.innerHTML = '';
+    for (let i = 0; i < items.length; i++) {
+      var html = itemTemplate(items[i]);
+      itemsContainer.innerHTML += html;
+    }
+  }
+}
+
+function filterByPrice(menu, price) {
+  let res = price.split(",");
+  maxPrice = parseInt(res[1]);
+  minPrice = parseInt(res[0]);
+  leftPrice.innerHTML = "&#36;" + minPrice;
```

Comment:
There is a shortcut for string interpolation (aka template literals): eg: \`whatever ${minPrice}`\ -> whatever 3
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals